src/Security/Authorization/Core/src/AuthorizationOptions.cs

```diff
@@ -14,7 +14,7 @@ public class AuthorizationOptions
         private Dictionary<string, AuthorizationPolicy> PolicyMap { get; } = new Dictionary<string, AuthorizationPolicy>(StringComparer.OrdinalIgnoreCase);
 
         /// <summary>
-        /// Determines whether authentication handlers should be invoked after a failure.
+        /// Determines whether authentication handlers should be invoked after <see cref="AuthorizationHandlerContext.HasFailed"/>.
```

Comment:
I looked up HasFailed and it doesn't say why it would fail.
https://github.com/dotnet/aspnetcore/blob/6f1b2cda6d4b81dbb8c924eaeef64d6604ff67bd/src/Security/Authorization/Core/src/AuthorizationHandlerContext.cs#L69-L72
HasFailed should be updated to say `if Fail has been called by any handler`.

Reply:
Sure that's the default implementation, but its virtual and the Context is pluggable, but those people hopefully know what they are doing